core/test/cljfmt/format/core_test.clj

```diff
@@ -23,6 +23,23 @@
          (reformat-string "(t/defrecord Foo\n [x]\nCloseable\n(close [_]\n(prn x)))"))))
 
 
+(deftest namespaced-maps
+  (is (= "#:a{:one 1\n    :two 2}"
+         (reformat-string "#:a{:one 1\n    :two 2}")))
+  (is (= "#:a{:one 1\n    :two 2}"
+         (reformat-string "#:a{:one 1\n    :two 2}" {:insert-missing-whitespace? false})))
+  (is (= "#:a {:one 1\n     :two 2}"
+         (reformat-string "#:a {:one 1\n     :two 2}")))
+  (is (= "#:a {:one 1\n     :two 2}"
```

Comment:
What happens when there is more than one space between the map namespace and the opening brace?

Reply:
It doesn't remove the space. Should I open up another issue for that?